roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: Enable extras repo
+  command: subscription-manager repos --enable=rhel-7-server-extras-rpms
+  when: inventory_hostname in groups.loadbalancers
+- name: Install docker
+  yum:
+    name: docker
+    state: latest
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: Enable and start docker daemon service
+  systemd:
+    name: docker
+    state: started
+    enabled: yes
+    daemon_reload: yes
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: create persistent storage volume for zabbix
+  command: docker run -d -v /var/lib/mysql --name zabbix-db-storage busybox:latest
+  when: inventory_hostname == 'localhost'
+  become: yes
+  become_method: sudo
+  args:
+    creates: .zabbix_persistence_created 
+- name: install zabbix db
+  command: docker run -d --name zabbix-db -v /backups:/backups -v /etc/localtime:/etc/localtime:ro --volumes-from zabbix-db-storage --env="MARIADB_USER=zabbix" --env="MARIADB_PASS=my_password" monitoringartist/zabbix-db-mariadb
```

Comment:
Is this a hardcoded password or am I reading it wrong?

Reply:
Fixed in latest commit